index.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Kuba Flashcard Ap</title>
```

Comment:
Our application is called 'Kuba Flashcard App'. Let's rename it here

Reply:
Of course